### ISSUE TITLE
Change dropdown to buttons for Income question on Eligibility Questionnaire

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#844]
Our new design includes a different form field type for the Income question on Eligibility Questionnaire. While the current site has a dropdown, we'd like to switch to a radio button. This means less step for users. We should also update the select options to make them more clear. The options should be as follows:

$0-$9,999
$10,000-$19,999
$20,000-$29,999
$30,000-$39,999
$40,000-$49,999
$50,000 or more

Design is viewable in Figma (03.5).

![eligibility_income_question_radio](https://user-images.githubusercontent.com/394908/145314232-92858198-f5e5-4c6f-a3b9-194b6f4e86ec.png)
